packages/react-components/lib/robots/robot-table-datagrid.tsx

```diff
@@ -42,22 +42,33 @@ export function RobotDataGridTable({ onRobotClick, robots }: RobotDataGridTableP
       const working = {
         color: theme.palette.success.main,
       };
+      const disabled = {
+        color: theme.palette.grey.A400,
```

Comment:
Does `theme.palette.action.disabled` or `theme.palette.action.disabledBackground` work instead?

Reply:
it works well, thanks! https://github.com/open-rmf/rmf-web/pull/920/commits/fa149f05c8020e0c137f4243b490da7bc40e200f